packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+export enum ReadingMode {
+  /**
+   * Do not read the content at all.
+   */
+  DoNotRead = 0,
+
+  /**
+   * The content in this component will read out after other reading tasks of screen reader.
+   * E.g. first read the information of currently focus control, then this content.
+   * NOTE: ReadingMode.ReadAfterOtherContent doesn't support ChromeVOX v53.
+   */
+  ReadAfterOtherContent = 1,
+
+  /**
+   * Read immediately after this component is triggered rendering and updated.
+   * Most screen readers will announce the word 'alert' before or after the whole sentence.
+   */
+  ReadImmediately = 2
+}
+
+export interface IScreenReaderAlertProps extends React.Props<HTMLElement> {
+  /**
+   * Use readingMode to specify whether to read the alert, or read before/after normal screen reader content.
+   *
+   * @default ReadingMode.ReadAfterOtherContent
+   */
+  readingMode?: ReadingMode;
+
+  /**
+   * Use indicator to specifically tell screen reader this text should be read even when text is not changed.
+   * Especially in case of re-reading the same text or the text that is possibly the same with the old one.
+   *
+   * Change the indicator to read the text no matter the text is changed or not.
+   * Keep indicator the same value or simply not use it to read ONLY when the text is changed.
+   */
+  indicator?: number;
```

Comment:
I would prefer the second option. Consumer might get confused when they using the string props while they can only use two or three options for that. They're not actually be able to pass any strings. By using enum as a type, consumer will know that they should import the enum **ReadingMode** and only be able to choose from it.